addons/mail/static/tests/discuss/core/web/messaging_menu.test.js

```diff
@@ -116,6 +116,9 @@ test("channel preview ignores transient message", async () => {
     await start();
     await openDiscuss(channelId);
     await insertText(".o-mail-Composer-input", "/who");
+    await contains(".o-mail-Composer-suggestion", { text: "who" });
+    await press("Enter");
+    await contains(".o-mail-Composer-suggestion", { count: 0 });
     await press("Enter");
```

Comment:
Nitpick: Enter selects the suggestion, 2nd enter is to send the command.
Not seeing the suggestion is more a symptom than the actual explicit intend that is to fill the composer content from the suggestion, and send when the composer is filled with the command.
I would prefer asserting the composer contains the command and then `press Enter`. This is more explicit in the actual intend in the test, therefore making it more maintainable.

Reply:
however, when i tested, evn if you have the `await contains(".o-mail-Composer-input", { text: "/who" });` you cannot guarantee that the suggestionList is closed. so that's why this line was added here.

Reply:
Ok for technical reason to await `count: 0` of suggestions, I assume this is to not have `ENTER` being handled by suggestion list.

I would add the assertion of composer content for functional clarity of intent of test though, but that's nitpicking ;)